src/jit/lsra.h

```diff
@@ -503,7 +503,7 @@ class LinearScan : public LinearScanInterface
                                       LSRA_ALWAYS_INSERT_RELOAD         = 0x400,
                                       LSRA_RELOAD_MASK                  = 0x400 };
     LsraReload                  getLsraReload()                 { return (LsraReload) (lsraStressMask & LSRA_RELOAD_MASK); }
-    bool                        alwaysInsertReload()            { return getLsraSpill() == LSRA_ALWAYS_INSERT_RELOAD; }
+    bool                        alwaysInsertReload()            { return getLsraReload() == LSRA_ALWAYS_INSERT_RELOAD; }
```

Comment:
It exposes another bug (only exposed by stress mode), for which I'm testing a fix now.  Meanwhile it's fine to check this in.